projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/case-flag/write-case-flag-field.component.ts

```diff
@@ -181,6 +171,32 @@ export class WriteCaseFlagFieldComponent extends AbstractFieldWriteComponent imp
     }
   }
 
+  public canMoveToFinalReviewStage(caseFlagState: CaseFlagState): boolean {
+    // If the journey is request support or manage support
+    // and the current state is either CaseFlagFieldState.FLAG_COMMENTS or CaseFlagFieldState.FLAG_UPDATE
+    // then move to final review stage
+    if (this.isDisplayContextParameterExternal) {
+      if (caseFlagState.currentCaseFlagFieldState === CaseFlagFieldState.FLAG_COMMENTS ||
```

Comment:
@johnbenjamin-hmcts - can probably re-write this as `return caseFlagState.currentCaseFlagFieldState === CaseFlagFieldState.FLAG_COMMENTS || caseFlagState.currentCaseFlagFieldState === CaseFlagFieldState.FLAG_UPDATE;`